packages/server/src/socket-server.ts

```diff
@@ -0,0 +1,67 @@
+import http from "http";
+import {
+  onOscJoinRequest,
+  onOscHostMessage,
+  onOscCtrlMessage,
+  onDisconnect,
+  onUserJoinRequest,
+} from "./socket";
+import { Socket, Server } from "socket.io";
+
+/**
+ * Initialize and setup socket server.
+ * @param server http.Server to attach socket.io server
+ * @param allowedOrigins array of urls for CORS
+ */
+export default function createSocketServer(server: http.Server, allowedOrigins: string[]) {
```

Comment:
the function body was simply cut-and-pasted from `server.ts`. The args are the only connection the socket-server had/has with the express server